app/views/questions/_form.html.erb

```diff
@@ -1,38 +1,74 @@
+<% if @question.errors.any? %>
+  <%= render "govuk_publishing_components/components/error_alert", {
+    message: "There were errors with your submission. Please fix them below."
+  } %>
+<% end %>
```

Comment:
Not something you need to do on this PR but something I thought I should flag.

On GOV.UK services we have a pattern of listing validation errors at the top of a page as well as on the individual field. It's explained somewhat in: https://design-system.service.gov.uk/components/error-summary/.

You can see how in GOV.UK Chat we've implemented this in: https://github.com/alphagov/govuk-chat/blob/16d27a1d821243d768e266b7c7235925808c23a4/app/views/sign_up/found_chat.html.erb#L7-L9 with the helper: https://github.com/alphagov/govuk-chat/blob/16d27a1d821243d768e266b7c7235925808c23a4/app/helpers/errors_helper.rb#L2-L7